tools/ldb_cmd.cc

```diff
@@ -1947,14 +1951,14 @@ void ReduceDBLevelsCommand::Help(std::string& ret) {
   ret.append("\n");
 }
 
-void ReduceDBLevelsCommand::OverrideBaseOptions() {
-  LDBCommand::OverrideBaseOptions();
-  options_.num_levels = old_levels_;
-  options_.max_bytes_for_level_multiplier_additional.resize(options_.num_levels,
+void ReduceDBLevelsCommand::OverrideBaseCFOptions(
+    ColumnFamilyOptions* cf_opts) {
+  cf_opts->num_levels = old_levels_;
```

Comment:
Do we want to still call the base class's `OverrideBaseCFOptions()` for option overrides common to `ldb` subcommands?

Reply:
Yes, good point!